model/version.go

```diff
@@ -628,6 +630,14 @@ func GetVersionsWithOptions(ctx context.Context, projectName string, opts GetVer
 		match[VersionRevisionOrderNumberKey] = revisionFilter
 	}
 
+	if !opts.CreatedAfter.IsZero() && !opts.CreatedBefore.IsZero() {
```

Comment:
nit: for queries built from user input, we generally use `utility.IsZeroTime` instead of the built-in `IsZero` because different languages use different zero times. Go uses midnight year 1 AD as the zero time, whereas many other languages like Python use the Unix epoch.